ScratchWikiSkin.skin.php

```diff
@@ -49,17 +49,32 @@ class SkinScratchWikiSkin extends SkinTemplate {
 		$template = 'ScratchWikiSkinTemplate', $useHeadElement = true;
 
 	/**
+	 * This function adds JavaScript via ResourceLoader
+	 *
+	 * Use this function if your skin has a JS file(s).
+	 * Otherwise you won't need this function and you can safely delete it.
+	 *
+	 * @param OutputPage $out
+	 */
+
+	public function initPage( OutputPage $out ) {
+		parent::initPage( $out );
+		$out->addModules('skins.scratchwikiskin2.js');
+		// make Chrome mobile testing work
+		$out->addMeta('viewport', 'user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width, height=device-height');
+	}
+
+		/**
 	 * Add CSS via ResourceLoader
 	 *
-	 * @param $out OutputPage
+	 * @param OutputPage $out
 	 */
-	public function setupSkinUserCss( OutputPage $out ) {
+	function setupSkinUserCss( OutputPage $out ) {
 		parent::setupSkinUserCss( $out );
-		$out->addModuleStyles( [
+		$out->addModuleStyles( array(
 			'mediawiki.skinning.interface', 'skins.scratchwikiskin2'
-		] );
-		// make Chrome mobile testing work
-		$out->addMeta('viewport', 'user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width, height=device-height');
+			/* 'skins.foobar' is the name you used in your skin.json file */
+		) );
```

Comment:
1. `[]` to `array()` is bad
2. didn't the issue literally say that the fact this call is in this function was the problem? You changed everything *but* what was needed.

Reply:
> 1. `[]` to `array()` is bad
> 2. didn't the issue literally say that the fact this call is in this function was the problem? You changed everything _but_ what was needed.

Sorry, I missed. Fixed the same problem again.